easybuild/easyconfigs/t/tbb/tbb-2021.9.0-GCCcore-12.2.0.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'CMakeNinja'
+
+name = 'tbb'
+version = '2021.9.0'
+
+homepage = 'https://github.com/oneapi-src/oneTBB'
+description = """Intel(R) Threading Building Blocks (Intel(R) TBB) lets you easily write parallel C++ programs that
+ take full advantage of multicore performance, that are portable, composable and have future-proof scalability."""
+
+toolchain = {'name': 'GCCcore', 'version': '12.2.0'}
+toolchainopts = {'cstd': 'c++11'}
```

Comment:
does this fail with the default c++17?

Reply:
This was first attempt to circumvent the above test error - it didn't work and I forgot to revert it.  I'll remove it.